packages/ui/src/kit/VInput/VInput.vue

```diff
@@ -59,24 +61,28 @@ export default {
       default: null,
     },
   },
+
   data() {
     return {
       isFocus: false,
     };
   },
+
   computed: {
     listeners() {
       return {
         ...this.$listeners,
-        input: value => {
-          this.$emit('input', value);
+        input: e => {
```

Comment:
can we remove as [this](https://github.com/endpass/endpass-core/pull/336/files#diff-6595f5ad3bdfd98e6096dee52754369eL30)